app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/ExportCollectionFragmentTest.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.sdp.tarjetakuna.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Environment
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasAction
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.R
+import com.github.sdp.tarjetakuna.model.MagicCard
+import com.github.sdp.tarjetakuna.model.MagicLayout
+import com.github.sdp.tarjetakuna.model.MagicSet
+import com.github.sdp.tarjetakuna.ui.collectionexport.ExportCollectionFragment
+import org.apache.poi.hssf.usermodel.HSSFWorkbook
+import org.junit.After
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.FileInputStream
+
+@RunWith(AndroidJUnit4::class)
+class ExportCollectionFragmentTest {
+
+    private lateinit var scenario: FragmentScenario<ExportCollectionFragment>
+
+    @Before
+    fun setUp() {
+        Intents.init()
+        scenario = launchFragmentInContainer()
+    }
+
+    @After
+    fun after() {
+        Intents.release()

```

Comment:
Intents.release() should be after the scenario.close() if you initialize it before scenario

Reply:
I fixed it, but if it is really a problem then watch out because it seems that you did the exact same in ScannerFragmentTest